action.yml

```diff
@@ -29,20 +29,31 @@ inputs:
     default: 'HEAD'
   api-version:
     description: 'SF API version to use during validate/deploy'
-    required: true
-    default: '51.0'
-  sfdx-download-url:
-    description: 'URL to download SFDX Linux client. Uses the XZ compressed version. Default is the stable version. Other version available at https://developer.salesforce.com/media/salesforce-cli/sfdx/versions/sfdx-linux-arm-tar-xz.json'
-    required: true
-    default: 'https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable/sfdx-linux-x64.tar.xz'
+    required: false
```

Comment:
I suggest not doing this...pinning API versions just like we do with the package-lock.json is preferred here for me.